### ISSUE TITLE
Capture browser console.log

### ISSUE THREAD

**Issue by datakurre** [robotframework/SeleniumLibrary#412]
Is it possible to capture console.log messages with Selenium and expose that to selenium2library? Dumping captured console.log would also be great addition to onError behavior.

**Comment by datakurre** [comment]
Selenium Wiki mentions plans for this and inclusion in "Wire" protocol. Woul you know, are those already implemented and usable through selenium library?

https://code.google.com/p/selenium/wiki/Logging
https://code.google.com/p/selenium/wiki/JsonWireProtocol#/session/:sessionId/log

**Comment by datakurre** [comment]
In addition, gettin JS stacktrace would be nice, if it's possible.

**Comment by datakurre** [comment]
It seems that Selenium Logging is implemented at least in Firefox WebDriver. It provides log types 'browser' and 'driver'. Yet, that command is not configured in python selenium bindings yet, but it must be manually configured.

```
def log_console(self):
    browser = self._current_browser()
    browser.command_executor._commands['log'] = (
        'POST', '/session/$sessionId/log'
    )
    return browser.execute('log', params={'type': 'browser'})['value']
```

Unfortunately, it seems that at least for FX this returns only 'Browser console' messages. Because it won't return messages from 'Web console' it's not very useful for debugging JS apps.

**Comment by just-be-dev** [comment]
I've done some investigation into this myself, but I haven't figured anything out yet. I'll let you know if I come up with something.

**Comment by aaltat** [comment]
I think this is now implemented by #863. If you disagree, please send an update.
